nc-games/src/utils/api.js

```diff
@@ -15,3 +15,9 @@ export const getReviewById = (review_id) => {
     return data.review;
   });
 };
+
+export const getCommentsByReviewId = (review_id) => {
+  return ncGamesApi.get(`/reviews/${review_id}/comments`).then(({ data }) => {
+    return data.comments;
```

Comment:
Great to see some destructuring in the API so your data doesnt need manipulating in the component 👍